.github/workflows/csharp-test.yml

```diff
@@ -104,21 +68,25 @@ jobs:
         # Rebuild only if we didn't get a precise cache hit.
         if: steps.cache-rust-deps.outputs.cache-hit == 'false'
         run: |
-          cargo install --force --path SpacetimeDB~/crates/cli --locked --message-format=short
-          cargo install --force --path SpacetimeDB~/crates/standalone --locked --message-format=short
+          cargo install --force --path crates/cli --locked --message-format=short
+          cargo install --force --path crates/standalone --locked --message-format=short
           # Add a handy alias using the old binary name, so that we don't have to rewrite all scripts (incl. in submodules).
           ln -sf $HOME/.cargo/bin/spacetimedb-cli $HOME/.cargo/bin/spacetime
         env:
           # Share the target directory with our local project to avoid rebuilding same SpacetimeDB crates twice.
-          CARGO_TARGET_DIR: unity-tests~/server-rust/target
+          CARGO_TARGET_DIR: demo/Blackholio/server-rust/target
 
       - name: Generate client bindings
-        working-directory: unity-tests~/server-rust
+        working-directory: demo/Blackholio/server-rust
         run: bash ./generate.sh -y
 
       - name: Check for changes
         run: |
-          git diff --exit-code unity-tests~/client-unity/Assets/Scripts/autogen || {
+          # This was copied from tools/check-diff.sh.
+          # It's required because `spacetime generate` creates lines with the SpacetimeDB commit
+          # version, which would make this `git diff` check very brittle if included.
+          PATTERN='^// This was generated using spacetimedb cli version.*'
+          git diff --exit-code --ignore-matching-lines="$PATTERN" -- demo/Blackholio/client-unity/Assets/Scripts/autogen || {
```

Comment:
the old `git diff` was essentially nonfunctional because `unity-tests~` was a submodule, so it always vacuously returned 0. Now that it's no longer a submodule, we had to fix up this check to work properly after #2673 .